changelog.d/pa-1659.changed

```diff
@@ -0,0 +1,14 @@
+`metavariable-comparison` now also works on metavariables that cannot be evaluated
+to simple literals. In such cases, we take the string representation of the code
+bound by the metariable. The way to access this string representation is via
+`str($MVAR)`. For example:
+
+```
+- metavariable-comparison:
+    metavariable: $X
```

Comment:
it's a bit weird/useless to have to specify metavariable:. Why $X and not $Y?
I think it was done back in the day by Matt Schwager but I think we could just remove it (or make
it optional to remain backward compatible).

Reply:
Done!